<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>860</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>860</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>  Maschine Learning</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: #151819;}</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="upperframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{background-color: #202124; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="regression">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: bold; color: #C1C1C1; text-align: center; border: None}
QPushButton:hover{background-color: rgba(255,255,255,0.1);}</string>
     </property>
     <property name="text">
      <string>Regression</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="classification">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: bold; color: #C1C1C1; text-align: center; border: None}
QPushButton:hover{background-color: rgba(255,255,255,0.1);}</string>
     </property>
     <property name="text">
      <string>Classification</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="deep_learn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {font: bold; color: #C1C1C1; text-align: center; border: None}
QPushButton:hover{background-color: rgba(255,255,255,0.1);}</string>
     </property>
     <property name="text">
      <string>Deep Learning</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="NoData">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>931</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{ color: rgba(255, 255, 255, 0.2); font: bold;  font-size: 50px;}</string>
    </property>
    <property name="text">
     <string>No Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="sideframePlaceHolder">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>31</y>
      <width>170</width>
      <height>830</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{   background-color:#0e1517;}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="sideframeNoData">
       <property name="styleSheet">
        <string notr="true">QLabel{ color: rgba(255, 255, 255, 0.2); font: bold;  font-size: 20px;}</string>
       </property>
       <property name="text">
        <string>No Data</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar{background-color: #202124;  color: #C1C1C1;}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_new_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDark_Mode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionThere_is_no_Help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar{ background-color: rgb(12, 12, 56)}</string>
   </property>
  </widget>
  <action name="actionOpen_new_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionDark_Mode">
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionThere_is_no_Help">
   <property name="text">
    <string>There is no Help</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
